### DISCUSSION TITLE
Add a margin in EarlyStoppingPlugin

### DISCUSSION THREAD

**Discussion by evertonaleixo** [ContinualAI/avalanche#990] [Feature Request]
I would like to add a new parameter to EarlyStoppingPlugin, called margin.

This parameter will be a float and will be used to define the minimal improvement to the plugin to accept the current step as a better model.

In some tests, the metric is improved by such a small amount, that does not really affect the accuracy model. I mean, sometimes, the improvement is just about 0.000001 in each epoch. Therefore, this parameter could reduce the training time of the experiment and still take advantage of the early stopping plugin.

I have already created a fork, implement this, and test. Do you think it's a good idea to merge into the principal repository?

Best regards,
Everton Aleixo

**Comment by AntonioCarta** [comment]
Basically, something like `min_delta` in [Keras](https://keras.io/api/callbacks/early_stopping/)? I think it would be a nice contribution, feel free to open a PR for this.

**Reply by evertonaleixo** [reply]
Yes!
Thank you!
